File: 2023/2023_04_04/2920.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] num = new int[8];

        for(int i = 0; i < 8; i++) {
            num[i] = scanner.nextInt();
        }

        if(num[0] == 1) {
            for(int i = 0; i < 8; i++) {
                if(num[i] != i+1) {
                    System.out.println("mixed");
                    return;
                }
            }
            System.out.println("ascending");
        } else if(num[0] == 8) {
            for(int i = 0; i < 8; i++) {
                if(num[i] != 8-i) {
                    System.out.println("mixed");
                    return;
                }
            }
            System.out.println("descending");
        } else {
            System.out.println("mixed");
        }

        scanner.close();
    }
}

File: 날짜별 (2023-2024.7)/2024/2024_07_06/2920.py
arr = list(map(int, input().split()))
ascending = True
descending = True

for i in range(8):
  if ascending and arr[i] != i + 1:
    ascending = False

  if descending and arr[i] != 8 - i:
    descending = False

if ascending:
  print('ascending')
elif descending:
  print('descending')
else:
  print('mixed')